git.c

```diff
@@ -546,6 +546,7 @@ static struct cmd_struct commands[] = {
 	{ "pickaxe", cmd_blame, RUN_SETUP },
 	{ "prune", cmd_prune, RUN_SETUP },
 	{ "prune-packed", cmd_prune_packed, RUN_SETUP },
+        { "psuh", cmd_psuh, RUN_SETUP },
```

Comment:
The Git project uses tabs, not spaces - that's why your whitespace looks weird here. I suggest you set your editor accordingly :)

Reply:
All right! Thanks. I saw that and went back to open the file and it looked normal. Makes more sense now. Thanks @nasamuffin

Reply:
i did git push -f Georgecanfly psuh to send updated branch in my fork. Now this says Outdated. Not sure if it's correct.